internal/project/framework/bootstrap/kernel.go

```diff
@@ -9,14 +9,15 @@ import (
 	"path/filepath"
 	"time"
 
-	"{{ .PkgPath }}/framework/dbdrivers"
-	"{{ .PkgPath }}/framework/internals/binder"
-	ierror "{{ .PkgPath }}/framework/internals/error"
-	"{{ .PkgPath }}/framework/internals/global"
-	"{{ .PkgPath }}/framework/internals/helpers"
-	imiddleware "{{ .PkgPath }}/framework/internals/middleware"
-	str "{{ .PkgPath }}/framework/internals/string"
-	"{{ .PkgPath }}/framework/internals/template"
+	/**#bean*/
+	"demo/framework/dbdrivers" /*#bean.replace("{{ .PkgPath }}/framework/dbdrivers")**/
```

Comment:
we should put these two lines in one line, right?

Reply:
`go fmt` command make it, `go fmt` will run before I commit

Reply:
yes, it works but not very beautiful... it's better to cancel `go fmt` before commit and push again.
@tanvir-retailai @yu-yk what do you guys think?